src/pages/jobs.js

```diff
@@ -138,50 +146,62 @@ const Jobs = () => {
 
   return (
     <Layout title="Jobs" largeTitle sidebar>
-      <h1>
-        Reactiflux <span>Jobs</span>
-      </h1>
-      <div>
-        {results.map((result, i) => (
-          <Offer
-            {...result}
-            key={result.id}
-            last={i === results.length - 1}
-            onClickGetInTouch={toggleModal}
-          />
-        ))}
-        <ResultsFooter>
-          {status === LOADING ? (
-            <p>Fetching posts...</p>
-          ) : next ? (
-            <Button onClick={onClickNext}>Fetch More Results</Button>
-          ) : null}
-          {status === ERROR ? (
+      {(setSidebar) => (
+        <>
+          <h1>
+            Reactiflux <span>Jobs</span>
+          </h1>
+          <FocusBoundary onChange={setSidebar}>
+            <Form
```

Comment:
Note that I've also moved these up above the main content, so that the sidebar comes before the content in the tab order.